Update version-control-basics.md (#5259)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/git/version-control-basics.md

## The .gitignore file

> dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) (ensuring these directories are not tracked) that automatically excludes the following sub-folders from your git repository to ensure smooth operation:

Author: ```suggestion
dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) that automatically excludes the following sub-folders from your git repository to ensure smooth operation:
```

> dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) that automatically excludes the following sub-folders from your git repository to ensure smooth operation. This global `.gitignore` is automatically applied to projects in dbt Cloud, ensuring these directories are not tracked, which is crucial for smooth dbt Cloud operation:

Author: ```suggestion
dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) (ensuring these directories are not tracked) that automatically excludes the following sub-folders from your git repository to ensure smooth operation:
```

```
dbt_packages/
logs/
target/
```

This inclusion uses a trailing slash, making these lines in the `.gitignore` file act as 'folder wildcards' that prevent any files or folders within them from being tracked by git. You can also specify additional exclusions as needed for your project.

> However, this global `.gitignore` _does not_ apply to dbt Core and dbt Cloud CLI users directly. Therefore, if you're working with dbt Core or dbt Cloud CLI, you need to manually add these three lines to your project's `.gitignore` file.

Author: ```suggestion
However, this global `.gitignore` _does not_ apply to dbt Core and dbt Cloud CLI users directly. Therefore, if you're working with dbt Core or dbt Cloud CLI, you need to manually add the three lines mentioned previously to your project's `.gitignore` file.
```
